opentreemap/treemap/lib/hide_at_zoom.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from math import floor
+
+from django.db import connection
+
+from treemap.models import MapFeature
+
+GRID_PIXELS = 2
+MAX_ZOOM = 14
+MIN_ZOOM = 0
+
+# When viewing a zoomed-out tree map, multiple plots in a local area are
+# rendered on top of one another. We set MapFeature.hide_at_zoom so the tiler
+# can render just one plot in such cases, speeding up tile creation for maps
+# with many plots.
+#
+# At each zoom level Z we find groups of plots located in the same cell of
+# a 2-pixel grid, and set hide_at_zoom = Z for all but one plot in each group.
```

Comment:
We may want this comment to refer to a  "GRID_PIXELS sized grid"

Is there a reason why we have chosen `GRID_PIXELS = 2`?